core/nginx/conf/nginx.conf

```diff
@@ -84,7 +84,11 @@ http {
       {% if WEBDAV_ADDRESS %}
       set $webdav {{ WEBDAV_ADDRESS }};
       {% endif %}
+      {% if MESSAGE_SIZE_LIMIT == 0 %}
```

Comment:
Have you tested this? I see that we cast it to "int" below...

Reply:
I just tested audioscaventer:patch-3 (by regenerating the webmail container) and it does not work.  The error I get adding a 17MB attachment (truncated at the ...) is:

`webmail_1    | 2022/01/31 19:37:58 [error] 13#13: *37 FastCGI sent in stderr: "PHP message: PHP Warning:  POST Content-Length of 17842503 bytes exceeds the limit of 8388608 bytes in Unknown on line 0" while reading response header from upstream, client: 192.168.203.2, server: , request: "POST /?/Upload/&q[]=/...`.

The problem is that in the webmail container even though /etc/nginx/http.d/rainloop.conf contains a `client_max_body_size 0;` the fact that /etc/nginx/nginx.conf has a `client_max_body_size 1m;` does not allow the post to succeed.  I know the 8388608 in the message is odd.  Maybe the error is in the fastcgi code.  Will look into that.

P.S. I have a solution (marioja:Mailu) where the value for client_max_body_size in /etc/nginx/http.d/rainloop.conf is set using the MESSAGE_SIZE_LIMIT as in the front container but I am getting the same error.

Reply:
If I set the `client_max_body_size 0;` in the /etc/nginx/nginx.conf, then I can upload the 17MB or even a 29MB file.  To fix this, we would need to modify the nginx.conf that is obtained via the apk add command for nginx.

Reply:
this proves that the `{% if MESSAGE_SIZE_LIMIT == 0 %}` test does not work since your limit is == 8388608 which is the problem i was trying to fix in the first place. 

Maybe we need to write `MESSAGE_SIZE_LIMIT|int == 0` or simply `MESSAGE_SIZE_LIMIT == '0'` instead. i am not familiar with the template tool's rules you use for those file, sorry

Reply:
I have **updated patch3** and _TESTED_ myself, this time. Good catch, thank you!!
Both `MESSAGE_SIZE_LIMIT|int == 0` and `MESSAGE_SIZE_LIMIT == '0'` work, but not `MESSAGE_SIZE_LIMIT == 0` that you tested, that's unfortunate.
Please confirm with updated version :)

The 1m default limit is overridden by whatever limit you set later in your conf, under server, http or location.
Below is a shot proving that i am indeed sending 66MB to gmail just fine with this 0 limit
![image](https://user-images.githubusercontent.com/3853962/151899172-9855e9a5-fdd5-401c-b7c7-90fdb10d370d.png)
and the reply from gmail refusing my 66mb email:
![image](https://user-images.githubusercontent.com/3853962/151899381-01a63cb8-e730-462e-883a-bdc97a087f80.png)

Reply:
I have never run any test with MESSAGE_SIZE_LIMIT set to 0.  I always used 50,000,000 (omit the commas for clarity).  When you talk about the problem you were trying to solve in the first place I believe you are referring to the problem of having a limit of 8,388,608 (again commas for clarity only) which only turn up if MESSAGE_SIZE_LIMIT is 0.